src/yggdrasil/core.go

```diff
@@ -180,7 +186,9 @@ func (c *Core) Start(nc *config.NodeConfig, log *log.Logger) (*config.NodeState,
 
 	c.config.Mutex.RLock()
 	if c.config.Current.SwitchOptions.MaxTotalQueueSize >= SwitchQueueTotalMinSize {
-		c.switchTable.queueTotalMaxSize = c.config.Current.SwitchOptions.MaxTotalQueueSize
+		phony.Block(&c.switchTable, func() {
```

Comment:
We might want to reverse this somewhat - that the switch takes the config mute and updates the `totalMaxSize` value itself when starting/reconfiguring, so that we don't need to do this here.